Dockerfile.amd64

```diff
@@ -30,30 +30,40 @@ RUN apk update && \
         libmount \
         libtbb@testing \
         libtbb-dev@testing \
-        opencv-dev@testing
+        opencv@testing \
+        opencv-dev@testing \
+       	libgomp \
+        libtbb@testing \
+        libtbb-dev@testing \
+        openblas \
+        openblas-dev
 
 WORKDIR /tmp
-ADD ./config.h /opt/sources/config.h
+# ADD ./include/config.h /opt/sources/config.h
 
+# Why do we have this if we have a similar cmake.Modules folder? Should we have a similar approach?
```

Comment:
This git pull to copy only one single is somewhat over-kill.

Reply:
Turns out that the file wasn't even needed.